js/containers/portal-dashboard/popup-class-nav.tsx

```diff
@@ -4,31 +4,34 @@ import { AnonymizeStudents } from "../../components/portal-dashboard/anonymize-s
 import { CustomSelect, SelectItem } from "../../components/portal-dashboard/custom-select";
 import { DashboardViewMode, FeedbackLevel, ListViewMode } from "../../util/misc";
 import { makeGetStudentFeedbacks } from "../../selectors/activity-feedback-selectors";
+import { getfeedbackSortRefreshEnabled } from "../../selectors/dashboard-selectors";
 import { CountContainer } from "../../components/portal-dashboard/count-container";
 import { SORT_BY_NAME, SORT_BY_MOST_PROGRESS, SORT_BY_LEAST_PROGRESS, SORT_BY_FEEDBACK_NAME,
-         SORT_BY_FEEDBACK_PROGRESS } from "../../actions/dashboard";
+         SORT_BY_FEEDBACK_PROGRESS, setFeedbackSortRefreshEnabled } from "../../actions/dashboard";
```

Comment:
I would've thought this should be `SET_FEEDBACK_SORT_REFRESH_ENABLED`. Is this correct though?

Reply:
I think it's correct to import `setFeedbackSortRefreshEnabled ` - it's the actual function that we need to call.  The other ones are the sort actions themselves that we tie directly to the dropdown menu.  Looking at it though, it seems like a weird route that was originally designed to handle the sorts.  Oh well - there is probably some logic behind it that I am not familiar with.